pinot-common/src/main/java/com/linkedin/pinot/pql/parsers/utils/JSONUtil.java

```diff
@@ -202,11 +201,6 @@ public Iterator keys() {
       return _inner.keySet().iterator();
     }
 
-    @Override
-    public Iterator sortedKeys() {
```

Comment:
Do you know what this impacts/do query results change?

Reply:
This function never gets called. I will merge after testing.

Reply:
I think this whole class can be removed (along with PQLParserUtils)